config/.example.single-seller-client-credentials.json

```diff
@@ -0,0 +1,47 @@
+{
+  "broker": {
+    "bookingPartners": {
+      "primary": {
+        "authentication": {
+          "initialAccessToken": null,
+          "clientCredentials": {
+            "clientId": "clientid_XXX",
+            "clientSecret": "example"
+          }
+        }
+      },
+      "secondary": {
+        "authentication": {
+          "initialAccessToken": null,
+          "clientCredentials": {
+            "clientId": "clientid_XXX",
+            "clientSecret": "example"
+          }
+        }
+      }
+    }
+  },
+  "integrationTests": {
+    "implementedFeatures": {
+      "multiple-sellers": false,
+      "booking-partner-authentication": false,
+      "dynamic-client-registration": false,
+      "business-to-consumer-tax-calculation-net": false,
+      "business-to-business-tax-calculation-net": false
+    }
+  },
+  "sellers": {
+    "primary": {
```

Comment:
Just a suggestion but I wonder if there needs to be either or both of the following:

* Assertion that if primary authentication style is client credentials then 1). there are no other sellers configured 2). the multiple-sellers features are disabled. Maybe this could be a test
* Documentation of Client Credentials mode in configuration which informs of this constraint

Reply:
Thanks @lukehesluke.
I have updated the broker documentation for Client Credentials, and have raised an issue here: https://github.com/openactive/openactive-test-suite/issues/520